Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -0,0 +1,301 @@
+// Copyright 2015~7 Dolphin Emulator Project
```

Comment:
What does the "~7" mean? From 2015 to 2017? Not sure that's the proper way to do so. Why not sticking with 2017?